src/Utilities/copy.jl

```diff
@@ -459,13 +425,142 @@ function default_copy_to(
     # Copy model attributes
     pass_attributes(dest, src, index_map)
     # Copy constraints
-    _pass_constraints(
-        dest,
-        src,
-        index_map,
-        constraints_not_added,
-        filter_constraints,
-    )
+    _pass_constraints(dest, src, index_map, constraints_not_added)
     final_touch(dest, index_map)
     return index_map
 end
+
+"""
+    ModelFilter(filter::Function, model::MOI.ModelLike)
+
+A layer to filter out various components of `model`. The filter function takes
+two arguments: an attribute (see below) and an element from the list returned by
+that attribute. It returns `true` if the element should be included in `dest`,
```

Comment:
`dest` is not defined in this context. I think `ModelFilter` can be defined outside the context of a `copy_to`. It could be also useful for printing a subset of a model for instance.